### PR TITLE
issue #3 resolved

### LINKED ISSUE #3: Current installer assumes extension directories
**Expected:**
_Prerequisite_:  **new** Ubuntu 19.04 LTS "Disco Dingo" OS distro installation
_GIVEN_ a cloned copy of the vpn indicator shell extension
  _AND_ the home user (i.e., not root)
  _AND_ a brand-new install of Linux distro
_WHEN_ I run the included `./install.sh`
_THEN_ the install should create the extension directory and install the vpn indicator successfully

**Actual:**
_Prerequisite_:  **new** Ubuntu 19.04 LTS "Disco Dingo" OS distro installation
_GIVEN_ a cloned copy of the vpn indicator shell extension
  _AND_ the home user (i.e., not root)
_WHEN_ I run the included `./install.sh`
_THEN_ the install script fails to remove the extension directory

**Problem Statement:** Currently, this app assumes that a user has already created the extension directory.  Specifically, the following line fails from this assumption:

https://github.com/howdoicomputer/vpn-indicator-shell-extension/blob/c538bad5b1428c41b57cd2795ddf717644c8cbe1/install.sh#L5

This results in a crashed install, even if run as root.  The solution should include a check for the directory existing before removing an old one.

### PR DESCRIPTION
**Pull request by amp5208**
resolves #3 

Tested on Ubuntu 19.04, new installation.  I hope you accept my PR :smile:

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,6 +1,14 @@
-#!/bin/bash
+#!/bin/bash -e
 
 dir=~/.local/share/gnome-shell/extensions/vpn-indicator@howdoicomputer.fastmail.com
 
-rm -rf $dir
+if [ -d $dir ]; then
```
Comment (amp5208) [inline install.sh:5]: i kept your remove, but put in a directory check - this ensures that bash does not fail out before attempting to remove a dir

File: install.sh [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+  rm -rf $dir
+else
+  echo "brand-new install..."
+fi
+mkdir -p $dir
```
Comment (amp5208) [inline install.sh:10]: the `-p` option here is equivalent to an `install` command, and ensures that the mkdir command does not fail out if a directory already exists.

File: install.sh [hunk 3]
```diff
@@ -1,6 +1,14 @@
-#!/bin/bash
+#!/bin/bash -e
```
Comment (amp5208) [inline install.sh:1]: i added a `-e` option here to your shell script to ensure that the script stops if a sigterm is ever received.

**Review by howdoicomputer — APPROVED** [review]

**Comment by howdoicomputer** [comment]
Looks good.

**Pull request merged**